dags/analyzer_helper/discord/discord_load_transformed_data.py

```diff
@@ -12,5 +14,6 @@ def __init__(self, platform_id: str):
     def load(self, processed_data: list[dict], recompute: bool = False):
         # Probably too aggressive, we should define another DAG for that
         if recompute:
+            logging.info("Recompute is true for loading discord transformed data, deleting the previous day")
```

Comment:
This is not deleting the previous day. It is deleting the whole data. You could write the following logs
```
recompute is true, deleting all the previous data!
```